metadata/lang/es.yml

```diff
@@ -52,21 +52,21 @@ license-framework: "GTN Framework tiene licencia del MIT" # "The GTN Framework i
 
 # slides
 thank-you: "Gracias" # Thank you!
-collaborative-effort: "TODO" # This material is the result of a collaborative work. Thanks to the Galaxy Training Network and all the contributors!
-edit-on-github: "TODO" # Found a typo? Something is wrong in this tutorial? Edit it on GitHub
-presenter-notes: "TODO" # Tip: press <kbd>P</kbd> to view the presenter notes
+collaborative-effort: ”Este material es resultado de trabajo colaborativo. ¡Agradecemos al Galaxy Training Network y a todos los contribuidores!” # This material is the result of a collaborative work. Thanks to the Galaxy Training Network and all the contributors!
+edit-on-github: “¿Encontraste algún error de escritura?,¿Existe algún error en este tutorial? Editalo en GitHub” # Found a typo? Something is wrong in this tutorial? Edit it on GitHub
+presenter-notes:”Consejo: presiona <kbd>P</kbd>  para visualizar las notas del presentador” # Tip: press <kbd>P</kbd> to view the presenter notes
 presenter-notes-explanation: |
-  "TODO
-
-  TODO (can be multi-line)"
+“Las notas del presentador contienen información extra la cual te puede ser de utilidad si vas a utilizar estas diapositivas para dar clases”
+“Presiona P de nuevo para ir a las notas del presentador”
+“Presiona C para crear una ventana nueva donde se visualizará la misma presentación”
+“Esta ventana está vinculada a la ventana principal. Si cambias las diapositivas en una causará que las diapositivas en la otra también cambien.”
+“Algo de utilidad cuando des una presentación”
 # Presenter notes contain extra information which might be useful if you intend to use these slides for teaching.
 #
 #Press P again to switch presenter notes off
 #
 #Press C to create a new window where the same presentation will be displayed.
 #This window is linked to the main window. Changing slides on one will cause the slide to change on the other.
-#
 #Useful when presenting.
-plaintext-slides: "TODO" # Or view the plaintext slides withouth JS
-before-diving-in: "TODO" # Before diving into this slide deck, we recommend you to have a look at
-
+plaintext-slides: “O visualiza las diapositivas con puro texto sin JS” # Or view the plaintext slides without JS
```

Comment:
```suggestion
plaintext-slides: “O visualiza las diapositivas en modo solo texto sin JS” # Or view the plaintext slides without JS
```

Reply:
Thanks @davelopez! Either you, @nomadscientist, or anybody else, feel free to hit "commit suggestion" button on these to accept them into the PR

Reply:
Sadly @shiltemann it's not giving me the option to 'commit suggestion'.  But every suggestion looks good!

Reply:
yeah no worries, it's because I had to touch those lines to update the quotation marks (yaml is picky about it), but will apply all the changes manually, thanks for reviewing!